src/audio/dai-zephyr.c

```diff
@@ -718,6 +718,15 @@ static int dai_set_dma_config(struct dai_data *dd, struct comp_dev *dev)
 		dma_block_cfg->block_size = config->elem_array.elems[i].size;
 		dma_block_cfg->source_address = config->elem_array.elems[i].src;
 		dma_block_cfg->dest_address = config->elem_array.elems[i].dest;
+		if (IS_ENABLED(CONFIG_DMA_NXP_EDMA)) {
```

Comment:
should this be a common thing we should always do ?
@abonislawski @kv2019i @lyakh pls comment.

Reply:
Looks like it should be safe to make this generic - Intel drivers don't use `.source_addr_adj` and `.dest_addr_adj`